bcgov/HOUS-permit-portal
# Update invitation flow UI

## Description

Update invitation flow UI

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

[BPHH-1229](https://hous-bssb.atlassian.net/browse/BPHH-1229)

## Steps to QA

1. Login as a super admin (reset DB if needed to prepopulate BCeID users) - BCeID: `super_admin` / `P@ssword1`
2. Invite a reviewer or review manager to a jurisdiction
3. Open the invitation link and note the updated UI
4. Verify the invited user can login with a new BCeID using the invitation link (Note - do not attempt to use an existing BCeID that is already tied to a user as this is not yet handled)

## UI Accessibility Checklist



_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?

n/a
